### DISCUSSION TITLE
Add .@add(x) .@mult(x) math modifiers to offset plots

### DISCUSSION THREAD

**Discussion by Combinacijus** [foxglove/community#425] [Q&A]
<https://foxglove.dev/docs/studio/panels/plot#math-modifiers>
![image](https://user-images.githubusercontent.com/16375702/200380365-cc0090eb-e916-46c3-9cf9-828a1cd84782.png)

**Comment by jtbandes** [comment]
Duplicate of https://github.com/orgs/foxglove/discussions/729 (the other one has a better description)
